molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/bootstrap/IndexBootstrapper.java

```diff
@@ -53,14 +55,15 @@ public void bootstrap()
 		else
 		{
 			LOG.debug("Index for AttributeMetaData found, index is present, no (re)index needed");
-			List<IndexJobExecution> failedIndexJobs = dataService.findAll(IndexJobExecutionMeta.INDEX_JOB_EXECUTION,
-					new QueryImpl<IndexJobExecution>().eq(JobExecutionMetaData.STATUS, "FAILED"),
+			List<IndexJobExecution> indexJobExecutions = dataService.findAll(IndexJobExecutionMeta.INDEX_JOB_EXECUTION,
+					new QueryImpl<IndexJobExecution>().eq(JobExecutionMetaData.STATUS, FAILED).or().eq(JobExecutionMetaData.STATUS, PENDING)
+							.or().eq(JobExecutionMetaData.STATUS, RUNNING).or().eq(JobExecutionMetaData.STATUS, CANCELED),
```

Comment:
You are right, RUNNING is not relevant but, it makes clear which status should be reindexed. Otherwise I should add comments by: https://github.com/molgenis/molgenis/blob/d6d355d9196ea28ed5924f18e0c6ba9b014c0db8/molgenis-jobs/src/main/java/org/molgenis/data/jobs/JobBootstrapper.java that changing this method influences the indexing bootstrap.
If I will change the code it can potentially introduce a bug in the future.